src/include/apparmor/aa-config.rb

```diff
@@ -33,20 +33,26 @@ def initialize_apparmor_aa_config(include_target)
       Yast.import "Label"
       Yast.import "Popup"
       Yast.import "Wizard"
+      Yast.import "Service"
     end
 
     def changeAppArmorState(aaEnabled)
       error = nil
       aaAction = ""
+      service = Yast::ServiceClass.new
 
       if aaEnabled == true
-        aaAction = "apparmor:enable"
+        error = service.start("apparmor")
+	if error == true
```

Comment:
bad indentation and not needed to test equal to true, so it can be `if error` but bigger problem is that it is in fact not error but success to `if success`